### PR TITLE
Fix rpm deps

### PR DESCRIPTION
**Pull request by oybed**
#### What does this PR do?
Changed order of installation, etc. to ensure correct versions of various packages - in particular "python-passlib", "ansible" and "jinja2". Thanks to @etsauer for assistance.

```
[root@8b4c40eea4da ~]# ansible --version
ansible 2.3.0.0
  config file = /root/.ansible.cfg
  configured module search path = Default w/o overrides
  python version = 2.7.5 (default, Nov  6 2016, 00:28:07) [GCC 4.8.5 20150623 (Red Hat 4.8.5-11)]
[root@8b4c40eea4da ~]# cat /usr/lib64/python2.7/site-packages/jinja2/__init__.py | grep '__version'
__version__ = '2.9.6'
[root@8b4c40eea4da ~]# rpm -qa | grep passlib                                                      
python-passlib-1.6.2-2.el7.noarch
```

#### How should this be manually tested?
Rebuild the docker control-host image and use it to run a full CASL ansible install

#### Is there a relevant Issue open for this?
N/A

#### Who would you like to review this?
cc: @redhat-cop/casl

### CODE REVIEW

**Comment by etsauer** [comment]
@oybed as part of this PR, can you also update docker-compose.yml to the following?

change:
```
image: redhatcop/openstack-client-centos
```
to
```
image: redhatcop/control-host-openstack
```

**Comment by oybed** [comment]
@etsauer updated with new name + updated README with instructions for how to set env variables, etc.

File: docker/control-host-openstack/README.md [hunk 1]
```diff
@@ -29,6 +29,23 @@ There are two options for running the `control-node` containers:
...
 ### Docker Compose (recommended)
 
+#### Environment Variables
+
+The following environment variables can be used to control the execution environment within the control host:
+
+| variable | purpose | default value |
+|:---------|:--------|:--------------|
+|SRC_CODE_DIR|The local source code directory|~/src|
+|DST_CODE_DIR|The remote directory for the source code directory|/root/code|
```
Comment (etsauer) [inline docker/control-host-openstack/README.md]: Can we say something besides 'remote' here? Destination inside the container? Something about the mount location?

File: docker/control-host-openstack/README.md [hunk 2]
```diff
@@ -29,6 +29,23 @@ There are two options for running the `control-node` containers:
...
+|SRC_SSH_DIR|The local ssh config directory|~/.ssh|
+|DST_SSH_DIR|The remote ssh config directory|/root/.ssh|
```
Comment (etsauer) [inline docker/control-host-openstack/README.md]: same

File: docker/control-host-openstack/README.md [hunk 3]
```diff
@@ -29,6 +29,23 @@ There are two options for running the `control-node` containers:
...
+|SRC_OSP_CONFIG_DIR|The local OpenStack config directory|~/.config/openstack|
+|DST_OSP_CONFIG_DIR|The remote OpenStack config directory|/root/.config/openstack|
```
Comment (etsauer) [inline docker/control-host-openstack/README.md]: same

File: docker/control-host-openstack/README.md [hunk 4]
```diff
@@ -29,6 +29,23 @@ There are two options for running the `control-node` containers:
...
+|SRC_ANSIBLE_CFG|The local Ansible config file|~/.ansible.cfg|
+|DST_ANSIBLE_CFG|The remote Ansible config file|/root/.ansible.cfg|
```
Comment (etsauer) [inline docker/control-host-openstack/README.md]: same

**Comment by oybed** [comment]
@etsauer updated per your comments + fixed a few other naming mis-alignments, i.e.: `control-host` instead of `control-node`. Can you please review and ensure this doesn't break anything else outside of the normal execution - i.e.: `docker.io`, etc.

**Comment by etsauer** [comment]
@oybed test running... I have one more small change I'd like to make:

```
[esauer@mgmt2 casl-ansible]$ git diff
diff --git a/docker/control-host-openstack/bin/start.sh b/docker/control-host-openstack/bin/start.sh
index 2da9d6c..1c01214 100755
--- a/docker/control-host-openstack/bin/start.sh
+++ b/docker/control-host-openstack/bin/start.sh
@@ -35,4 +35,6 @@ if [ "$1" == "daemonize" ];then
   shift
 fi
 
+echo "export TERM=xterm" >> /root/.bashrc
+
 exec "$@"

```

**Comment by etsauer** [comment]
@oybed test ran successfully

**Pull request merged**